Gems/PhysX/Code/Source/PhysXCharacters/Components/CharacterControllerComponent.cpp

```diff
@@ -586,30 +589,39 @@ namespace PhysX
                 sceneInterface->RegisterSceneSimulationStartHandler(m_attachedSceneHandle, m_sceneSimulationStartHandler);
             }
         }
+
+        AZ::TransformNotificationBus::Handler::BusConnect(GetEntityId());
+        Physics::CharacterRequestBus::Handler::BusConnect(GetEntityId());
+        Physics::CollisionFilteringRequestBus::Handler::BusConnect(GetEntityId());
+        AzPhysics::SimulatedBodyComponentRequestsBus::Handler::BusConnect(GetEntityId());
```

Comment:
nit: Could extract `EntityId` to a local to stop having to call this function repeatedly

```
const AZ::EntityId entityId = GetEntityId();
AZ::TransformNotificationBus::Handler::BusConnect(entityId);
... // etc
```